app/controllers/spree/api/v2/storefront/feedback_reviews_controller.rb

```diff
@@ -0,0 +1,83 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        class FeedbackReviewsController < ::Spree::Api::V2::ResourceController
+          before_action :load_review, only: :create
+          before_action :require_spree_current_user if Spree::Reviews::Config[:require_login]
+
+          def index
+            render_serialized_payload { serialize_collection(paginated_collection) }
+          end
+
+          def create
+            if @review.present?
+              @feedback_review = @review.feedback_reviews.new(feedback_review_params)
+              @feedback_review.user = spree_current_user
+              @feedback_review.locale = I18n.locale.to_s if Spree::Reviews::Config[:track_locale]
```

Comment:
can you please add TODO: in this line 

we want to check if it is right to let user disable track_locale, there has been some new changes in recent spree regarding locale and we want to ensure we don't break reviews features in newer spree projects

Reply:
done dai